framework/src/schema/application_config_schema.ts

```diff
@@ -357,6 +382,89 @@ export const applicationConfigSchema = {
 				},
 			},
 		},
+		genesis: {
```

Comment:
some of the props should go into specific module?

Reply:
Yes these should go, but may be @shuse2  plan is to do it with the issues for each module when we start.